pkg/nuctl/client/api.go

```diff
@@ -70,6 +72,21 @@ func NewNuclioAPIClient(parentLogger logger.Logger,
 		return nil, errors.New("Access key must be provided")
 	}
 
+	// Ensure that apiURL is a correct URL
+	baseURL, err := url.Parse(apiURL)
+	if err != nil {
+		return nil, errors.New(fmt.Sprintf("Failed to parse API URL. API URL: %s", apiURL))
+	}
+
+	// Ensure that given API URL contains /api suffix
+	// Add it if it doesn't
+	if !strings.Contains(baseURL.Path, "api") {
```

Comment:
Might be better to check the suffix instead of "contains". (what does `baseURL.Path` return?) 
e.g. `https://api.com/api/something`